CV-HTML-CSS/style.css

```diff
@@ -0,0 +1,149 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  height: 100%;
+}
+
+body {
+  height: inherit;
+  font-family: "Lato", Arial, serif !important;
+}
+
+h2 {
+  color: rgb(56, 56, 56);
+}
+
+.contact-wrapper {
+  display: flex;
+  height: 100%;
+  width: 100%;
+}
+.contact-image {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  height: 100%;
+  width: 50%;
+  background: linear-gradient(90deg, rgba(2, 2, 2, 0.8), rgba(26, 24, 24, 0.8)),
+    url("https://images.unsplash.com/photo-1526925539332-aa3b66e35444?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=701&q=80")
+      no-repeat center center / cover;
+  object-fit: contain;
+}
+.info {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 230px;
+  margin: 20px 0;
+}
+.info i {
+  color: white;
+  font-size: 30px;
+  padding-right: 20px;
+  text-shadow: 2px 2px rgb(133, 133, 133);
+}
+.info span {
+  text-transform: uppercase;
+  font-size: 30px;
+  color: #25b9ca;
+  margin-bottom: 10px;
+  display: inline-block;
+}
+
+.info p {
+  font-size: 18px;
+  font-style: italic;
+
+  color: #cacaca;
+}
+
+.right-part {
+  color: black;
+  background-color: #ffffff;
+background-image: linear-gradient(315deg, #ffffff 0%, #d7e1ec 74%);
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  align-items: center;
+  justify-content: flex-start;
+
+  position: relative;
+}
+
```

Comment:
upper u have duplicate code and I would suggest u to refactor it in the next way
```
.contact-image,
.right-part {
    display: flex;
    align-items: center;
    flex-direction: column;
    flex: 1; // the only purpose u use width: 50%; in this rules is to divide screen in 2 the same width parts and u can
    do it with use of flex: 1;
}

.contact-image {
    justify-content: center;
    ...
}

.right-part {
    justify-content: flex-start;
    ...
}
```
 firstly u write common part and then parts, that differ

 also write rules, that relative to block model (display, width, height) first and relative to text
(color, background) last;